openapi-generator/src/main/kotlin/org/jellyfin/openapi/builder/openapi/OpenApiApiServicesBuilder.kt

```diff
@@ -13,17 +15,16 @@ import org.jellyfin.openapi.constants.MimeType
 import org.jellyfin.openapi.constants.Security
 import org.jellyfin.openapi.constants.Strings
 import org.jellyfin.openapi.hooks.ApiTypePath
+import org.jellyfin.openapi.hooks.DefaultValueHook
 import org.jellyfin.openapi.hooks.ServiceNameHook
-import org.jellyfin.openapi.model.ApiService
-import org.jellyfin.openapi.model.ApiServiceOperation
-import org.jellyfin.openapi.model.ApiServiceOperationParameter
-import org.jellyfin.openapi.model.HttpMethod
+import org.jellyfin.openapi.model.*
```

Comment:
Doesn't the style enforce no wildcards?

Reply:
yes but intellij/android studio doesn't

Reply:
Well, you can turn them off in the IDE, but unfortunately it's not that reliable..

Reply:
If the EditorConfig file can't change those I won't use it, because not all contributors will.

Reply:
I know, I was mostly talking about this case, since it's an unnecessary change and we could at least enforce it in reviews and for ourselves, even if it doesn't work for all contributors by default.

Reply:
Fair enough, I think in this case it changed to wildcard because it now uses 5>= imports from the model package